docker-compose.debug.yml

```diff
@@ -17,7 +17,7 @@ services:
       SPRYKER_XDEBUG_HOST_IP: ${SPRYKER_XDEBUG_HOST_IP}
       PHP_IDE_CONFIG: ${PHP_IDE_CONFIG}
     env_file:
-        - .env.prod
+        - docker/.env
```

Comment:
please, move to the infrastructure folder as we already keep docker-related files there.